ci/kokoro/ubuntu/build.sh

```diff
@@ -16,13 +16,25 @@
 
 set -eu
 
+echo "================================================================"
+echo "Running Bazel build with integration tests against production $(date)."
+echo "================================================================"
+
 echo "Reading CI secret configuration parameters."
 source "${KOKORO_GFILE_DIR}/test-configuration.sh"
 
 echo "Running build and tests"
 cd "$(dirname $0)/../../.."
 readonly PROJECT_ROOT="${PWD}"
 
+echo "================================================================"
+echo "Update or Install Bazel $(date)."
+echo "================================================================"
+"${PROJECT_ROOT}/ci/install-bazel.sh" linux
+
+export PATH=$HOME/bin:$PATH
```

Comment:
Wouldn't it be safer to:
```sh
BAZEL_BIN=$HOME/bin/bazel
```
and then just use `${BAZEL_BIN}` instead of `bazel`?

Reply:
Yup. Fixed (and rebased).